### ISSUE TITLE
Make sure to avoid "orphan records"

### ISSUE THREAD

**Issue by payne-chris-r** [ga-wdi-boston/rails-validations-constraints#10]
If Lists have many Items. If Items belong to a List, you need to make sure that List has a Item (just like how we would make sure that each item had a primary key).

**Comment by gaand** [comment]
This doesn't seem correct to me.  Why can't a list exist without an item?  

Examples:  any newly created list; a todo list with all items completed and deleted.

**Comment by payne-chris-r** [comment]
Yeah, @gaand it was just a note to myself. Basically saying that "items" belong to "lists." They're related and therefore can't be created without belonging to a list. A list can certainly exist without items, but not hte other way around.
